pkg/controller/provider/web/vsphere/workload.go

```diff
@@ -112,6 +113,10 @@ func (r *Workload) Expand(db libmodel.DB) (err error) {
 	}
 	err = db.Get(host)
 	if err != nil {
+		log.Error(
```

Comment:
Could we instead of logging inside the inventory send a error response status to the main?